protocol/socketio.go

```diff
@@ -50,6 +50,8 @@ func Encode(msg *Message) (string, error) {
 		return "", err
 	}
 
+	result = result + msg.Namespace
```

Comment:
Namespace is just appended to the msg type string?

Are there any protocol reference docs you could tag here?

Reply:
Nope. I guessed by printing out the message sent by Python socketio client.

Reply:
😂